src/components/pathway-progress/PathwayProgress.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+import PathwayProgressPage from './PathwayProgressPage';
+import './styles/index.scss';
```

Comment:
I don't think we really need this import here.

Reply:
Also, can we discard this component and directly call `PathwayProgressPage` in enterprise page routes? Unsure why we need this extra hierarchy here.

Reply:
Removed `PathwayProgress`, we need the `scss` import as without it the styles are not imported.